daemon/src/cfd_actors.rs

```diff
@@ -116,9 +114,8 @@ pub async fn handle_oracle_attestation(
 
     for id in db::load_all_cfd_ids(&mut conn).await? {
         let cfd = load_cfd(id, &mut conn).await?;
-        let event = try_continue!(cfd
-            .decrypt_cet(&attestation)
-            .context("Failed to decrypt CET using attestation"));
+
+        let event = cfd.decrypt_cet(&attestation)?;
```

Comment:
I think we should keep the `try_continue` here. Just because one CFD fails doesn't mean we should abort the loop.